core/sequencer/server_test.go

```diff
@@ -229,6 +229,7 @@ func TestHighWatermarks(t *testing.T) {
 	fakeLogs, idx := setupLogs(ctx, t, dirID, map[int64]int{0: 10, 1: 20})
 	s := Server{logs: fakeLogs}
 
+	quantum := time.Microsecond
```

Comment:
Would these tests break if it was a `Nanosecond`?

Reply:
The mysql implementation returns `high` + 1 microsecond, the memory implementation returns `high` + 1 nanosecond.  This PR standardized on microsecond since it was the higher granularity.  I'd like to remove this complexity though.